modules/Staff/staff_manage_edit.php

```diff
@@ -53,22 +57,16 @@
             echo __('You have not specified one or more required parameters.');
             echo '</div>';
         } else {
-            try {
-                $data = array('gibbonStaffID' => $gibbonStaffID);
-                $sql = 'SELECT gibbonStaff.*, title, surname, preferredName, initials, dateStart, dateEnd FROM gibbonStaff JOIN gibbonPerson ON (gibbonStaff.gibbonPersonID=gibbonPerson.gibbonPersonID) WHERE gibbonStaffID=:gibbonStaffID';
-                $result = $connection2->prepare($sql);
-                $result->execute($data);
-            } catch (PDOException $e) {
-                echo "<div class='error'>".$e->getMessage().'</div>';
-            }
+            $staffGateway = $container->get(StaffGateway::class);
+            $staff = $staffGateway->selectStaffByStaffID($gibbonStaffID);
 
-            if ($result->rowCount() != 1) {
+            if ($staff->isEmpty()) {
                 echo "<div class='error'>";
                 echo __('The specified record cannot be found.');
                 echo '</div>';
             } else {
                 //Let's go!
-                $values = $result->fetch();
+                $values = $staff->toDataSet()->toArray()[0];
```

Comment:
In this instance $staff is a result object, so we can still call fetch on it rather than converting to and from a data set :)
```suggestion
                $values = $staff->fetch();
```